eox_tagging/test/test_models.py

```diff
@@ -0,0 +1,202 @@
+"""
+Test classes for Tags model
+"""
+
+
+from django.contrib.auth.models import User
+from django.core.exceptions import ObjectDoesNotExist, ValidationError
+from django.test import TestCase, override_settings
+
+from eox_tagging.models import Tag
+from eox_tagging.test_utils import CourseFakeModel, EnrollmentsFakeModel
+
+
+@override_settings(EOX_TAGGING_DEFINITIONS=[{"field_name": "tag_value",
```

Comment:
You can write this in the settings test file

Reply:
I did, @felipemontoya told me to change it as a recommendation

Reply:
so... @felipemontoya  why here instead of the test settings file ?

Reply:
Because we want to change the settings for every test